moduleroot/CONTRIBUTING.md

```diff
@@ -156,24 +157,32 @@ If you already have those gems installed, make sure they are up-to-date:
 % bundle update
 ```
 
+## Running Tests
+
 With all dependencies in place and up-to-date we can now run the tests:
 
+### Unit Tests
+
+With the PDK (recommended):
+```shell
+% pdk test unit
+```
+
+Without the PDK:
+
 ```shell
 % bundle exec rake spec
 ```
 
-This will execute all the [rspec tests](http://rspec-puppet.com/) tests
-under [spec/defines](./spec/defines), [spec/classes](./spec/classes),
-and so on. rspec tests may have the same kind of dependencies as the
-module they are testing. While the module defines in its [Modulefile](./Modulefile),
+This will execute all the [rspec tests](http://rspec-puppet.com/) in the directories defined [here](https://github.com/puppetlabs/puppetlabs_spec_helper/blob/699d9fbca1d2489bff1736bb254bb7b7edb32c74/lib/puppetlabs_spec_helper/rake_tasks.rb#L17) and so on. 
+rspec tests may have the same kind of dependencies as the module they are testing. While the module defines these dependencies in its [metadata.json](./metadata.json),
```

Comment:
Change "While" to "Although"

There's nothing wrong with "while" but it can be misunderstood by non-native English speakers, so I sort of shy away from it